devices/config/device.go

```diff
@@ -20,10 +20,10 @@ type Device struct {
 	FileMode os.FileMode `json:"file_mode"`
 
 	// Uid of the device.
-	Uid uint32 `json:"uid"`
+	Uid uint32 `json:"uid,omitempty"` //nolint:revive
```

Comment:
Can you add a comment what the `nolint` was for?


```suggestion
	Uid uint32 `json:"uid,omitempty"` //nolint:revive // Ignore false positive that the pod bay door was open
```

Reply:
I actually tried but failed to come up with a nice comment. Something like '// Too late to change public API IDs.` does not sound good.

Reply:
Came up with something; PTAL @thaJeztah